FHEM/00_SIGNALduino.pm

```diff
@@ -1348,6 +1348,14 @@ sub SIGNALduino_CheckVersionResp {
 			$hash->{logMethod}->($name, 5, "$name: CheckVersionResp, cc1101 available");
 			SIGNALduino_Get($hash, $name,"ccconf");
 			SIGNALduino_Get($hash, $name,"ccpatable");
+		} else {
+			# connect device without cc1101 to port where a device with cc1101 was previously connected (example DEF with /dev/ttyUSB0@57600) #
+			$hash->{logMethod}->($hash, 5, "$name: CheckVersionResp, delete old READINGS from cc1101 device");
+			delete($hash->{cc1101_available}) if defined( $hash->{cc1101_available} );
```

Comment:
Gibt leider Abzüge in der Critic Bewertung, ich bin auch der Meiung, uns interessiert ob es existiert nicht unbedingt ob des defined ist: 
```
if exists $hash->{cc1101_available} {
delete $hash->{cc1101_available}  
}
```